plugins/Compressor/CompressorControlDialog.h

```diff
@@ -214,6 +220,8 @@ private slots:
 	PixmapButton * feedbackButton;
 	PixmapButton * lookaheadButton;
 
+	bool guiVisibility;
```

Comment:
```suggestion
	bool m_guiVisibility = true;
```
Coding convention for data members.

Also moved the initialization to here instead of in the constructor.